connect-openvpn.py

```diff
@@ -95,25 +122,37 @@ def main():
     # Read configurations from config.ini
     try:
         config.read_file(open('config.ini'))
+
+        logging.basicConfig(level=logging.DEBUG)
+        logger_h = logging.FileHandler(config['general']['LOG_FILE'])
+        logger_h.setFormatter(logging.Formatter('%(name)s - %(funcName)20s() - %(asctime)s - %(levelname)s - %(message)s'))
+        logger.addHandler(logger_h)
+
     except Exception as e:
-        print("Could not read config.ini file")
+        logger.error("Could not read config.ini file")
```

Comment:
Should we use logger.error("", exc_info=True) or logger.exception("") to print the traceback to the log?

Reply:
Maybe it would be good to log the trace also